tests/tests_password.yml

```diff
@@ -58,34 +63,49 @@
       meta: flush_handlers
 
     - name: >
-        Include role to change the password to the existing password.
-        Should report Not Changed.
+        Include role to change the password.
+        Should report Changed.
       include_role:
         name: linux-system-roles.mssql
       vars:
         mssql_password: "p@55w0rD"
 
+    - meta: flush_handlers
```

Comment:
all tasks should have `name:` - otherwise ansible-lint or ansible-test will complain

Reply:
Done.